From 4f42f449bf5611d1ebdc6bb8ceff4cb073c6ddf7 Mon Sep 17 00:00:00 2001
From: AndroPlus <hosigumayuugi@gmail.com>
Date: Sun, 20 Feb 2022 15:18:43 +0900
Subject: [PATCH 1/1] Fix changelog error

---
 build/tools/changelog.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/tools/changelog.sh b/build/tools/changelog.sh
index 05f860f7..15b403c4 100755
--- a/build/tools/changelog.sh
+++ b/build/tools/changelog.sh
@@ -56,5 +56,7 @@ done
 
 sed -i 's/project/   */g' $Changelog
 
+Changelog=$(readlink -f $Changelog)
+
 cp $Changelog $OUT_DIR/target/product/$DEVICE/system/etc/
 mv $Changelog $OUT_DIR/target/product/$DEVICE/
-- 
2.25.1

